lib/Functional/DropWhile.ark

```diff
@@ -0,0 +1,12 @@
+(let dropWhile (fun (f L) {
+    (mut n 0)
+    (mut output [])
+    (mut continue true)
+
+    (while (and (< n (len L)) continue) {
+        (if (= true (f (@ L n)))

```

Comment:
the condition can be simplified in `(if (f (@ L n)) ...)` because the if construction automatically check if the condition value is truthy

Reply:
I could have recalled this not working, but I just tried it and it worked. So I've made the change. I must have been doing something wrong when I first tried it.